tests/nodes_test.h

```diff
@@ -189,6 +189,16 @@ void TestNodeFindError() {
   UastFree(ctx);
 }
 
+void TestEmptyResult() {
+  NodeIface iface = IfaceMock();
+  Uast *ctx = UastNew(iface);
+  Node module = Node("Module");
+
+  CU_ASSERT_FATAL(UastFilter(ctx,
+              &module, "//Import[@roleImportDeclaration]//alias") == NULL);
```

Comment:
Incorrect indentation.

Reply:
Mmmm, I don't see anything regarding continuation of long lines in Google's C++ coding style guide, aligning with the opening parenthesis is fine? (I see another line where it's done like that):

```cpp
  CU_ASSERT_FATAL(UastFilter(ctx, &module,
                             "//Import[@roleImportDeclaration]//alias") == NULL);
```

Reply:
Right, it should be aligned: https://google.github.io/styleguide/cppguide.html#Function_Calls